File: URIS2023-AgilIS-Tim10-main/BacklogItem_MicroService/BacklogItem_MicroService/Models/Entities/Description.cs
using System.ComponentModel.DataAnnotations;

namespace BacklogItem_MicroService.Models.Entities
{
    public class Description
    {
        [Key]
        public Guid DescriptionId { get; set; }
        [Required]
        [MaxLength(200)]
        public string DescriptionText { get; set; }
        public Guid BacklogItemId { get; set; }
        public BacklogItem BacklogItem { get; set; }

       
    }
}

File: URIS2023-AgilIS-Tim10-main/URIS2023_TempateProject-main/src/app/models/description.ts
import { BacklogItem } from "./backog-item"

export class Description {
   descriptionId: string
   descriptionText: string
   backlogItemId: string
   backlogItem: BacklogItem
}

export class DescriptionCreation {
    descriptionText: string
     backlogItemId: string

   constructor(descriptionText, backlogItemId ){
    this.descriptionText=descriptionText;
    this.backlogItemId =backlogItemId
   }
}

export class DescriptionUpdate {
    descriptionId: string
    descriptionText: string

    constructor(descriptionId, descriptionText ){
      this.descriptionId =descriptionId
      this.descriptionText=descriptionText;
       }

 }